src/ManagedServices/ManagedServices/Commands/RemoveAzureRmManagedServicesDefinition.cs

```diff
@@ -30,15 +30,10 @@ public class RemoveAzureRmManagedServicesDefinition : ManagedServicesCmdletBase
     {
         protected const string DefaultParameterSet = "Default";
         protected const string ByInputObjectParameterSet = "ByInputObject";
-        protected const string ByResourceIdParameterSet = "ByResourceId";
 
-        [Parameter(ParameterSetName = DefaultParameterSet, Mandatory = true, HelpMessage = "The registration definition identifier.")]
```

Comment:
removing mandatory parameter is breaking change and is not allowed in non-breaking change release